keyboards/keebio/quefrency/keymaps/marianas/keymap.h

```diff
@@ -0,0 +1,33 @@
+#include QMK_KEYBOARD_H
+
+#undef DEBOUNCE
+#define DEBOUNCE 15
```

Comment:
This actually needs to go in a `config.h` file.  If it's not there, it won't get processed correctly.